spec/features/irs_attempts_api/event_tracking_spec.rb

```diff
@@ -150,29 +150,4 @@
       expect(received_event_types).to match_array(expected_event_types)
     end
   end
-
-  # rubocop:disable Layout/LineLength
-  scenario 'reset password from an IRS with new browser session and without request_id does not track event' do
-    freeze_time do
-      user = create(:user, :signed_up)
-      visit_idp_from_ial1_oidc_sp(
-        client_id: service_provider.issuer,
-        irs_attempts_api_session_id: 'test-session-id',
-      )
-
-      visit root_path
-      fill_forgot_password_form(user)
-      set_new_browser_session
-      click_reset_password_link_from_email
-      fill_reset_password_form(without_request_id: true)
```

Comment:
I could use a confidence check on this, but I had a hard time following what this was testing for, since it seemed to be handling a case where someone would manually edit the hidden input value in the page markup? That's not something we care to support, is it? 

If we wanted, I could update this to test if a user were to reset password with a different, non-IRS service provider?

cc @olatifflexion via git blame

Reply:
Agreed with @mitchellhenke  since the form doesn’t have a parameter with these changes it is fine to remove the test, it was intended to test if someone manually edits the hidden input value in the page.